src/sw/Core/Inc/SensorObserver.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _SENSOROBSERVER_H_
+#define _SENSOROBSERVER_H_
+
+class SensorObserver
```

Comment:
Or Listener as I have written before, can be observer but in both places

Reply:
change to Listener